api/v1beta1/types.go

```diff
@@ -229,6 +229,11 @@ type LoadBalancerSpec struct {
 	ID string `json:"id,omitempty"`
 	// +optional
 	Name string `json:"name,omitempty"`
+	// +optional
+	FrontendIPs []FrontendIP `json:"frontendIPs,omitempty"`
```

Comment:
Note to other reviewers: This was moved out of `LoadBalancerClassSpec` since it is more cluster specific.